web/app.js

```diff
@@ -323,6 +325,9 @@ var PDFViewerApplication = {
     });
     this.pdfSidebar.onToggled = this.forceRendering.bind(this);
 
+    this.pageNumberInput = document.getElementById('pageNumber');
```

Comment:
Sure, but this patch has already been bit-rotten a couple of times, and until I get an OK on the feature itself *and* the general implementation, it doesn't make much sense for me to put in the time required to rebase this yet again.
Getting past the `viewer.js` -> `app.js` rename was "fun" enough :-)